main.tf

```diff
@@ -323,11 +323,12 @@ module "cache" {
 ################################################################################
 resource "aws_iam_instance_profile" "instance" {
   name = "${local.name_iam_objects}-instance"
-  role = aws_iam_role.instance.name
+  role = local.aws_iam_role_instance_name
   tags = local.tags
 }
 
 resource "aws_iam_role" "instance" {
+  count                = var.create_runner_iam_role ? 1 : 0
```

Comment:
question: do we need a separate `create_runner_iam_role` or can we use something like `var.runner_aim_role_name > ""` instead?

Reply:
Thanks for the feedback.

As users may have differing IAM requirements, I think both are useful to support possible use cases:

 - Default / current behaviour: Create role and use default suggested name.
 - Create role with custom name: `runner_iam_role_name=myrole`
 - Do not create role, use some existing role: `runner_iam_role_name=myrole` and `create_runner_iam_role=false`

Reply:
Oh yes, true.